### ISSUE TITLE
VxKex enabled programs under sandbox - Sandboxie compatibility

### ISSUE THREAD

**Issue by ask2018** [vxiiduu/VxKex#134]
I've been trying to enable Sandboxie  - https://github.com/sandboxie-plus/Sandboxie/ for some programs (Brave Portable and Infekt) running with VxKex enabled, but Sandboxie crashes in such case. My question is it is possible to make VxKex compatible with Sandboxie? If so, I'm also not sure if such thing must be done from VxKex or Sandboxie side.

**Comment by vxiiduu** [comment]
No, it seems like Sandboxie is rather invasive inside the target applications, and I'm not interested in making any major changes to get it to work. You have to choose one or the other.

**Comment by ask2018** [comment]
Thank you for answer and I understand you decision. I’ll try to ask Sandboxie devs, if there is something they can/want to do for the compatibility from their side. For now the critical applications I need to run in Sandbox (for security reasons) are still compatible with Win7, but this might change in the near future (and in such case unfortunately choosing one or the other will be not a good option), that’s why I tried this and asked this question.

**Comment by DavidXanatos** [comment]
@vxiiduu Given that you are not using Application verifier for anything more than to get the dll loaded, 
I would suggest to try to use sandboxies dll injection mechanism instead
```
InjectDll64=[path to 64 bit]\VxKex.dll
InjectDll=[path to 32 bit]\VxKex.dll
```
If the fault is caused by the Application Verifier mechanism then that would solve this issue, although VxKex DLL would need to be modified to do its thing on DLL_PROCESS_ATTACH instead of DLL_PROCESS_VERIFIER.

Its a small change and worth a try.
